tlf-disposition.Rmd

```diff
@@ -1,41 +1,43 @@
 # Disposition {#disposition}
 
 Following [ICH E3 guidance](https://database.ich.org/sites/default/files/E3_Guideline.pdf), 
-we need to summarize the accounting of all participants who entered the study, 
-in Section 10.1, Disposition of Participants.
+we need to summarize the accounting of all participants who entered the study in Section 10.1, Disposition of Participants.
```

Comment:
a summary needs to be provided to account for

Reply:
```suggestion
A summary needs to be provided to account for of all participants who entered the study in Section 10.1, Disposition of Participants.
```